server/controllers/profile.js

```diff
@@ -0,0 +1,53 @@
+const Profile = require("../models/ProfileModel");
+
+// create new profile using given arguments
+const addProfile = async (props) => {
+  let newProfile = await new Profile(props);
+  let profile = await newProfile.save();
+  return profile;
+};
+exports.createProfile = async (req, res, next) => {
+  try {
+    let response = await addProfile(req.body);
```

Comment:
Make sure to destructure the required `Profile` fields from the request body and also validate them.

Reply:
for destruction, I don't know exactly what is the keys inside the request.body.
for validation i think it will be in the front end because there is yup and formik in the package json of the client

Reply:
It's a good practice to validate data on the server side as well in case someone tries to send requests to our API manually.